raiden/benchmark/speed_decoding.py

```diff
@@ -39,76 +44,84 @@ def test_ack(iterations=ITERATIONS):
 
 def test_ping(iterations=ITERATIONS):
     msg = Ping(nonce=0)
-    msg.sign(PRIVKEY)
+    msg.sign(PRIVKEY, ADDRESS)
     run_timeit('Ping', msg, iterations=iterations)
 
 
-# def test_rejected(iterations=ITERATIONS):
-#     msg = Rejected(HASH, 1, [])
-#     msg.sign(PRIVKEY)
-#     run_timeit('Rejected', msg, iterations=iterations)
-# def test_rejected_with_args(iterations=ITERATIONS):
-#     msg = Rejected(HASH, 1, [1, 2, 3])
-#     run_timeit('Rejected with args', msg, iterations=iterations)
+# TODO: LOCKSROOT_REJECTED
 
 
 def test_secret_request(iterations=ITERATIONS):
+    identifier = 1
     hashlock = HASH
     msg = SecretRequest(
-        1,  # TODO: fill in identifier
+        identifier,
         hashlock
     )
-    msg.sign(PRIVKEY)
+    msg.sign(PRIVKEY, ADDRESS)
     run_timeit('SecretRequest', msg, iterations=iterations)
 
 
 def test_secret(iterations=ITERATIONS):
+    identifier = 1
     secret = HASH
     msg = Secret(
-        1,  # TODO: fill in identifier
+        identifier,
         secret
     )
-    msg.sign(PRIVKEY)
+    msg.sign(PRIVKEY, ADDRESS)
     run_timeit('Secret', msg, iterations=iterations)
 
 
 def test_direct_transfer(iterations=ITERATIONS):
+    identifier = 1
     nonce = 1
     asset = ADDRESS
     balance = 1
     recipient = ADDRESS
     locksroot = HASH
 
-    msg = DirectTransfer(nonce, asset, balance, recipient, locksroot)
-    msg.sign(PRIVKEY)
-    run_timeit('DirectTransfer', msg, iterations=iterations)
-
-
-def test_locked_transfer(iterations=ITERATIONS):
-    amount = 1
-    expiration = 1
-    hashlock = sha3(ADDRESS)
-    lock = Lock(amount, expiration, hashlock)
-
-    nonce = 1
-    asset = ADDRESS
-    balance = 1
-    recipient = ADDRESS
-    locksroot = sha3(ADDRESS)
-    msg = LockedTransfer(
-        1,  # TODO: fill in identifier
+    msg = DirectTransfer(
+        identifier,
         nonce,
         asset,
         balance,
         recipient,
         locksroot,
-        lock
     )
-    msg.sign(PRIVKEY)
-    run_timeit('LockedTransfer', msg, iterations=iterations)
+    msg.sign(PRIVKEY, ADDRESS)
+    run_timeit('DirectTransfer', msg, iterations=iterations)
+
+
+# LockedTransfer cannot be encoded/decoded because it's not sent throught the
```

Comment:
Looks fine to me. Only thing is that if LockedTransfer is not to be tested why not just remove this from here?